src/__test__/components/main/Input.test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+*/
+import React from 'react'
+
+import { render, fireEvent, waitFor, screen } from '@testing-library/react'
+import Input from "../../../components/main/Input"; 
+
+const setup = () => {
+    const utils = render(<Input />)
+    const input = utils.getByLabelText('input-date')
+    return {
+        input,
+        ...utils,
+    }
+}
```

Comment:
instead of a custom function it could be changed to use a `beforeEach` and the input could be tested using the `screen`

```jsx
describe("Header Component", () => {
  beforeEach(() => {
    render(<Input />);
  })

  test("must display date on format YYYY-MM-DD", () => {
    const input = screen.getByLabelText('input-date')
    fireEvent.change(input, { target: { value: '1993-03-14' } })

    expect(input.value).toMatch(/^\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$/)
  })
});
```